brave/brave-core
# Removes user wallet UI for JP

Resolves https://github.com/brave/brave-browser/issues/5883

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- make sure  that you see links in the panel and uphold button
- close browser
- open Preferences file and search for `countryid_at_install`
- replace current value with `19024`
- start browser again
- make sure that panel looks like this
![image](https://user-images.githubusercontent.com/9574457/65015170-0c003880-d921-11e9-84d3-8674a3421abf.png)
![image](https://user-images.githubusercontent.com/9574457/65015177-0e629280-d921-11e9-8089-fd0d5c17e572.png)


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
